app/scripts/utils/colours.es6

```diff
@@ -10,3 +10,35 @@ export default {
   poor: '#f05e55',
   unknown: '#7d7d7d',
 };
+
+export const Color = {};
+
+Color.getWaterpointColor = (status) => {
+  let color = null;
+  switch (status) {
+  case 'FUNCTIONAL':
+    color = 'green';
```

Comment:
yes, indeed, but I did't not know if they are going to be red, yellow and green (on the mock up they are blue, light blue and grey) moreover the Non Functional have different categories.